### PR TITLE
add api.loads and api.dumps

### PR DESCRIPTION
**Pull request by djmitche**
Regular old `json.dumps` will barf on a WSME type; and `json.loads` will successfully load such a thing, but represent it as a dictionary rather than a WSME object.  These little gems take care of both problems.

### CODE REVIEW

File: relengapi/docs/development/api_methods.rst [hunk 1]
```diff
@@ -34,11 +34,27 @@ See the WSME_ documentation for more detail.
...
+
+        print(api.dumps(JsonThing, my_thing))
+
+.. py:function:: relengapi.lib.api.loads(datatype, obj)
+
+    Load the given object from a JSON string, assuming it is of the given WSME datatype.
+    For example::
+
+        api.loads(JsonThing, data)
+
```
Comment (Callek) [inline relengapi/docs/development/api_methods.rst]: Can you expend the examples a bit to have "input" and "output" results in the example. Reading this doc without the code (below) is pretty confusing to me what this expects/yields.

Comment (Callek) [inline relengapi/docs/development/api_methods.rst]: Just putting in-place something akin to what you have in the test should be enough.

**Comment by Callek** [comment]
:+1: with some doc clarity possibly as suggested

**Comment by djmitche** [comment]
Better?

**Comment by Callek** [comment]
:+1:

**Pull request merged**